package/features/steps/cli_steps.py

```diff
@@ -80,6 +80,29 @@ def create_project_from_config_file(context):
     assert res.returncode == OK_EXIT_CODE
 
 
+@given("I have run a non-interactive kedro new with {starter} starter")
```

Comment:
Does the starter name really matter or can we just say `with starter`? I think that's what we went with on kedro & kedro-docker.

Reply:
Yea it doesn't really matter. Funny that was what caused problems for me... I used "pandas-iris" in the test and it didn't work and I couldn't figure out why 🤦  should have just hard coded.